src/category_theory/category.lean

```diff
@@ -114,10 +114,10 @@ end
 
 variables (α : Type u)
 
-instance [preorder α] : small_category α :=
-{ hom  := λ U V, ulift (plift (U ≤ V)),
-  id   := λ X, ⟨ ⟨ le_refl X ⟩ ⟩,
-  comp := λ X Y Z f g, ⟨ ⟨ le_trans f.down.down g.down.down ⟩ ⟩ }
+instance preorder_category [preorder α] : category.{0} α :=
```

Comment:
Drop this?

Reply:
Dropped.

Reply:
It's still here though.

Reply:
Sorry, too many branches. :-) I've tried again.